src/components/CustomHeader.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import "../styles/App.css";
+
+function CustomHeader({ text }) {
```

Comment:
How do you think we should implement the navigation buttons? As a separate component or in CustomHeader and then hide or show them depending on the screen?

Reply:
We will add it in CustomHeader component